src/pipelines.js

```diff
@@ -531,6 +692,9 @@ class AutomaticSpeechRecognitionPipeline extends Pipeline {
         for (let aud of audio) {
             aud = await this._preprocess(aud, sampling_rate)
 
+            /**
+             * @type {any}
```

Comment:
```suggestion
             * @type {any[]}
```

Could be at least an "any array"?